code/modules/jobs/job_types/quartermaster.dm

```diff
@@ -19,14 +19,16 @@
 	display_order = JOB_DISPLAY_ORDER_QUARTERMASTER
 	bounty_types = CIV_JOB_RANDOM
 	departments = DEPARTMENT_CARGO
+	family_heirlooms = list(/obj/item/stamp, /obj/item/stamp/denied)
+	mail_goodies = list(
+		/obj/item/circuitboard/machine/emitter = 3
```

Comment:
bruh

Reply:
c'mon, there's gotta be some better loot you can put in here

Reply:
Quartermasters don't have access to printing their own emitters.
The Emitter crate requires engineering access to open.
Emitters as you know are highly sought after by cargo since they can break locks.
This enables them a way to bypass the other requirements by themselves, rarely.